### PR TITLE
Add MPS release-notes/ciflow tags

### PR DESCRIPTION
**Pull request by malfet**

### CODE REVIEW

File: torchci/lib/bot/autoLabelBot.ts [hunk 1]
```diff
@@ -113,9 +115,11 @@ const notUserFacingPatternExceptions: RegExp[] = [
 // ]
 const repoSpecificAutoLabels: {[repo: string]: [RegExp, string][]}  = {
   // Sample entry below
```
Comment (ZainRizvi) [inline torchci/lib/bot/autoLabelBot.ts:117]: nit: let's get rid of this comment

File: torchci/lib/bot/autoLabelBot.ts [hunk 2]
```diff
@@ -113,9 +115,11 @@ const notUserFacingPatternExceptions: RegExp[] = [
...
-  // "pytorch/pytorch": [
-  //    [/torch\/(csrc|cpp)\/api/gi, "release notes: cpp"]
-  // ],
+  "pytorch/pytorch": [
+      [/aten\/src\/ATen\/mps/gi, , "ciflow/mps"],
+      [/aten\/src\/ATen\/native\/mps/gi, , "ciflow/mps"],
+      [/test\/test_mps.py/gi, , "ciflow/mps"],
```
Comment (janeyx99) [inline torchci/lib/bot/autoLabelBot.ts:121]:
```suggestion
      [/aten\/src\/ATen\/mps/gi, "ciflow/mps"],
      [/aten\/src\/ATen\/native\/mps/gi, "ciflow/mps"],
      [/test\/test_mps.py/gi, "ciflow/mps"],
```
copy paste mistake?

also could you add a test case :)

File: torchci/lib/bot/autoLabelBot.ts [hunk 3]
```diff
@@ -113,9 +115,11 @@ const notUserFacingPatternExceptions: RegExp[] = [
 // ]
 const repoSpecificAutoLabels: {[repo: string]: [RegExp, string][]}  = {
   // Sample entry below
-  // "pytorch/pytorch": [
-  //    [/torch\/(csrc|cpp)\/api/gi, "release notes: cpp"]
-  // ],
+  "pytorch/pytorch": [
+      [/aten\/src\/ATen\/mps/gi, , "ciflow/mps"],
```
Comment (ZainRizvi) [inline torchci/lib/bot/autoLabelBot.ts:119]: Extra comma causing errors here
```suggestion
      [/aten\/src\/ATen\/mps/gi, "ciflow/mps"],
```

**Review by ZainRizvi — CHANGES_REQUESTED** [review]
Looking good, just needs one bugfix

File: torchci/lib/bot/autoLabelBot.ts (hunk 2 above)
Comment (malfet) [reply torchci/lib/bot/autoLabelBot.ts]: I'm not sure we want to test repo specific rules here

**Review by ZainRizvi — APPROVED** [review]

**Pull request merged**